src/node_http2.h

```diff
@@ -491,22 +491,22 @@ class SessionShutdownWrap : public ReqWrap<uv_idle_t> {
                       uint32_t errorCode,
                       int32_t lastStreamID,
                       Local<Value> opaqueData,
-                      bool immediate,
```

Comment:
Don't forget to clean up the docs. They mention the `immediate` option which seems to be deprecated here.